runtime/doc/vimfn.txt

```diff
@@ -7561,6 +7561,19 @@ prompt_getprompt({buf})                                     *prompt_getprompt()*
                 Return: ~
                   (`any`)
 
+prompt_gettext({buf})                                         *prompt_gettext()*
```

Comment:
Reviewing `prompt_getprompt`, I think that `prompt_getinput` is a better name after all. Because `prompt_getprompt` vs `prompt_gettext` is very ambiguous. 

I will also update the docs for `prompt_getprompt` since "prompt text" is very ambiguous.

p.s. I don't really get why `prompt_getprompt` is needed at all. Especially if `':` works properly, `prompt_getprompt` just serves no purpose.

Reply:
let's go with `prompt_getinput`. 

> p.s. I don't really get why prompt_getprompt is needed at all. Especially if ': works properly, prompt_getprompt just serves no purpose.

I think `prompt_getprompt` is basically a pair of `prompt_setprompt` as getter/setter. Currently, ': gives the start of the prompt not the end of it. Even if we are marking the end of the prompt, it'd require test manipulation where this function directly returns it. Since it's already in for quite a while, I don't see much reason to break it ¯\_(ツ)_/¯